roles/calibre/tasks/main.yml

```diff
@@ -1,62 +1,68 @@
-- name: Get Calibre setup file (CentOS)
-# the installer works for intel fedora, and Centos, and deals with dependencies
-  get_url:
-    url="{{ calibre_src_url }}"
-    dest="{{ downloads_dir }}/calibre-installer.py"
-    mode=0755
-  when: ansible_distribution == "CentOS"
+- name: Check if /library/calibre/metadata.db exists
+  stat:
+    path: "{{ calibre_dbpath }}/metadata.db"
+  register: calibre_db
 
-- name: Install Calibre (CentOS)
-  shell: "{{ downloads_dir }}/calibre-installer.py >> /dev/null"
-  args:
-    creates: /usr/bin/calibre-uninstall
-  when: calibre_install and ansible_distribution == 'CentOS'
+- name: Installing Calibre via calibre-installer.py
+  include_tasks: installing.yml
```

Comment:
Should this be `include_tasks: installer.yml` (or should the corresponding file be `installing.yml` ?)

Reply:
yup